From 52b8277142e6bfb0f6fad6e0c493a8a810d07e1c Mon Sep 17 00:00:00 2001
From: Hao Bui <hao.bui.yg@renesas.com>
Date: Wed, 21 Oct 2020 10:13:14 +0700
Subject: [PATCH 246/475] ARM: dts: r8a7742-iwg21d-q7-dbcm-ca: Change to use
 vin1_clk_b pin group

The iWave RZ/G1H board (iwg21m) use vin1_clk_b pin group instead of
vin1_clk. The current pin setting (vin1_clk) made conflict with
EthernetAVB.

Signed-off-by: Binh Nguyen <binh.nguyen.uw@renesas.com>
Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
---
 arch/arm/boot/dts/r8a7742-iwg21d-q7-dbcm-ca.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/r8a7742-iwg21d-q7-dbcm-ca.dts b/arch/arm/boot/dts/r8a7742-iwg21d-q7-dbcm-ca.dts
index 059c163..cc3c852 100644
--- a/arch/arm/boot/dts/r8a7742-iwg21d-q7-dbcm-ca.dts
+++ b/arch/arm/boot/dts/r8a7742-iwg21d-q7-dbcm-ca.dts
@@ -195,7 +195,7 @@
 	};
 
 	vin1_pins: vin1 {
-		groups = "vin1_data8", "vin1_clk";
+		groups = "vin1_data8", "vin1_clk_b";
 		function = "vin1";
 	};
 
-- 
2.7.4

